azure-mgmt-website/src/main/java/com/microsoft/azure/management/website/WebApp.java

```diff
@@ -194,7 +168,7 @@
      */
     String defaultHostName();
 
-    List<HostNameBinding> hostNameBindings() throws CloudException, IOException;
+    List<HostNameBinding> getHostNameBindings();
```

Comment:
This one, however, seems possible. There is probably only one binding from one host to one web app.